website/content/apis/_index.md

```diff
@@ -2,985 +2,768 @@
-<a href="https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#secretreference-v1-core">
-Kubernetes core/v1.SecretReference
-</a>
-</em>
-</td>
-<td>
-<em>(Optional)</em>
-<p>passwordSecret is name of the authentication secret for BGP Peer.
-the secret must be of type &ldquo;kubernetes.io/basic-auth&rdquo;, and created in the
-same namespace as the MetalLB deployment. The password is stored in the
-secret as the key &ldquo;password&rdquo;.</p>
-</td>
-</tr>
-<tr>
-<td>
-<code>bfdProfile</code><br/>
-<em>
-string
-</em>
-</td>
-<td>
-<em>(Optional)</em>
-<p>The name of the BFD Profile to be used for the BFD session associated to the BGP session. If not set, the BFD session won&rsquo;t be set up.</p>
-</td>
-</tr>
-<tr>
-<td>
-<code>ebgpMultiHop</code><br/>
-<em>
-bool
-</em>
-</td>
-<td>
-<em>(Optional)</em>
-<p>To set if the BGPPeer is multi-hops away. Needed for FRR mode only.</p>
-</td>
-</tr>
-<tr>
-<td>
-<code>vrf</code><br/>
-<em>
-string
-</em>
-</td>
-<td>
-<em>(Optional)</em>
-<p>To set if we want to peer with the BGPPeer using an interface belonging to
-a host vrf</p>
-</td>
-</tr>
-</table>
-</td>
-</tr>
-<tr>
-<td>
-<code>status</code><br/>
-<em>
-<a href="#metallb.io/v1beta2.BGPPeerStatus">
-BGPPeerStatus
-</a>
-</em>
-</td>
-<td>
-</td>
-</tr>
-</tbody>
-</table>
-<hr/>
-<p><em>
-Generated with <code>gen-crd-api-reference-docs</code>
-.
-</em></p>
+# API Reference
+
+## Packages
+- [metallb.io/v1alpha1](#metallbiov1alpha1)
+- [metallb.io/v1beta1](#metallbiov1beta1)
+- [metallb.io/v1beta2](#metallbiov1beta2)
+
+
+## metallb.io/v1alpha1
+
+Package v1alpha1 contains API Schema definitions for the metallb v1alpha1 API group
+
+### Resource Types
+- [AddressPool](#addresspool)
+- [AddressPoolList](#addresspoollist)
+
+
+
+#### AddressPool
+
+
+
+AddressPool is the Schema for the addresspools API.
+
+_Appears in:_
+- [AddressPoolList](#addresspoollist)
+
+| Field | Description |
+| --- | --- |
+| `apiVersion` _string_ | `metallb.io/v1alpha1`
+| `kind` _string_ | `AddressPool`
+| `kind` _string_ | Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds |
+| `apiVersion` _string_ | APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources |
+| `metadata` _[ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v/#objectmeta-v1-meta)_ | Refer to Kubernetes API documentation for fields of `metadata`. |
+| `spec` _[AddressPoolSpec](#addresspoolspec)_ |  |
+| `status` _[AddressPoolStatus](#addresspoolstatus)_ |  |
+
+
+#### AddressPoolList
```

Comment:
Let's ignore lists

Reply:
This is how you do that:

https://github.com/metallb/frr-k8s/pull/66/commits/21875cda65c2dc47310317d1d62d2f4cd2cddad9#diff-0bd1bfb29c091e48cde3b1490429624af7ed53a468b72e1370d0d236162b09b7R4

Reply:
Done!